tests/queryEditor.spec.ts

```diff
@@ -135,3 +135,35 @@ test('"Timeseries custom query" multiple ts OK', async ({ selectors, readProvisi
   const tsWithUnits = tsExternalIds.map((ts, i) => `${ts}-${units[i]}`);
   await expect(panelEditPage.panel.fieldNames).toContainText(tsWithUnits);
 });
+
+
+test('"Event query" as table is OK', async ({ page, gotoDashboardPage, readProvisionedDashboard, grafanaVersion }) => {
+  const dashboard = await readProvisionedDashboard({ fileName: 'weather-station.json' });
+  const dashboardPage = await gotoDashboardPage(dashboard);
+  
+  const panelEditPage = await dashboardPage.gotoPanelEditPage('3')
+
+  await expect(panelEditPage.panel.fieldNames).toContainText(["externalId", "description", "startTime", "endTime"]);
+
+  const query = `
+    {
+      "prefix": {
+          "property": ["externalId"],
+          "value": "test_event (1"
+      }
+    }`;
+
+  if (semver.gte(grafanaVersion, '10.2.0')) {
+    await page.getByTestId(/Code editor container/).getByRole("textbox").first().fill(query, { force: true });
```

Comment:
does it not work without `force: true` ?

Reply:
yes, you are right. fixed now